`make format` fails on Linux

The `resolve_container_path.py` script exits with an exception when run outside of a docker container on Linux:

```
$ make format
docker run \
    --attach stdout \
    --rm \
    --volume $(python scripts/resolve_container_path.py /home/nadove/PycharmProjects/azul):/home/developer/azul \
    --workdir /home/developer/azul \
    $(AZUL_DEBUG=0 python -m azul 'docker.resolve_docker_image_for_launch("pycharm")') \
    /opt/pycharm/bin/format.sh -r -settings .pycharm.style.xml -mask '*.py' src scripts test lambdas/layer/app.py lambdas/indexer/app.py lambdas/service/app.py .flake8/azul_flake8.py environment.py deployments/anvilbox/environment.py deployments/anvildev.browser/environment.py deployments/anvildev/environment.py deployments/anvildev.gitlab/environment.py deployments/anvildev.shared/environment.py deployments/anvilprod.browser/environment.py deployments/anvilprod/environment.py deployments/anvilprod.gitlab/environment.py deployments/anvilprod.shared/environment.py deployments/dev.browser/environment.py deployments/dev/environment.py deployments/dev.gitlab/environment.py deployments/dev.shared/environment.py deployments/hammerbox/environment.py deployments/nadove2.local/environment.py deployments/nadove4.local/environment.py deployments/nadove5.local/environment.py deployments/nadove6.local/environment.py deployments/nadove.local/environment.py deployments/prod.browser/environment.py deployments/prod/environment.py deployments/prod.gitlab/environment.py deployments/prod.shared/environment.py deployments/sandbox/environment.py deployments/tempdev.browser/environment.py deployments/tempdev/environment.py deployments/tempdev.gitlab/environment.py deployments/tempdev.shared/environment.py .github/workflows/schedule.py terraform/data_sources.tf.json.template.py terraform/sqs.tf.json.template.py terraform/api_gateway.tf.json.template.py terraform/providers.tf.json.template.py terraform/step_function.tf.json.template.py terraform/common.tf.json.template.py terraform/backend.tf.json.template.py terraform/authentication.tf.json.template.py terraform/cloudwatch.tf.json.template.py terraform/dynamo.tf.json.template.py terraform/s3.tf.json.template.py terraform/elasticsearch.tf.json.template.py terraform/lambda_layer.tf.json.template.py terraform/gitlab/gitlab.tf.json.template.py terraform/shared/shared.tf.json.template.py terraform/browser/browser.tf.json.template.py lambdas/indexer/.chalice/config.json.template.py lambdas/service/.chalice/config.json.template.py .github/pull_request_template.md.template.py resources/public_sources.json.template.py resources/environ.json.template.py
2026-02-11 23:54:28,959    INFO MainThread __main__: Trying to extract ID of current container from /proc/self/mountinfo
Traceback (most recent call last):
  File "/home/nadove/PycharmProjects/azul/scripts/resolve_container_path.py", line 74, in <module>
    main(sys.argv[1])
    ~~~~^^^^^^^^^^^^^
  File "/home/nadove/PycharmProjects/azul/scripts/resolve_container_path.py", line 68, in main
    host_path = resolve_container_path(container_path)
  File "/home/nadove/PycharmProjects/azul/scripts/resolve_container_path.py", line 56, in resolve_container_path
    container_id = one(container_ids)
  File "/home/nadove/PycharmProjects/azul/.venv/lib/python3.13/site-packages/more_itertools/more.py", line 627, in one
    raise too_short or ValueError('too few items in iterable (expected 1)')
ValueError: too few items in iterable (expected 1)
2026-02-11 23:54:29,075    INFO MainThread azul.docker: Resolving 'pycharm' image TagImageRef(registry='docker.io', username='ucscgi', repository=('azul-pycharm',), tag='2025.2.6-72') …
2026-02-11 23:54:29,075    INFO MainThread azul.docker: Resolved 'pycharm' image to DigestImageRef(registry='122796619775.dkr.ecr.us-east-1.amazonaws.com', username='docker.io', repository=('ucscgi', 'azul-pycharm'), digest='sha256:ee7398068ca13ac564300c33b391ac0bd3909af3537a3ab741bdc797733376bd')
2026-02-11 23:54:29,075    INFO MainThread azul.docker: Pulling image DigestImageRef(registry='122796619775.dkr.ecr.us-east-1.amazonaws.com', username='docker.io', repository=('ucscgi', 'azul-pycharm'), digest='sha256:ee7398068ca13ac564300c33b391ac0bd3909af3537a3ab741bdc797733376bd') …
2026-02-11 23:54:30,030    INFO MainThread azul.docker: Pulled image DigestImageRef(registry='122796619775.dkr.ecr.us-east-1.amazonaws.com', username='docker.io', repository=('ucscgi', 'azul-pycharm'), digest='sha256:ee7398068ca13ac564300c33b391ac0bd3909af3537a3ab741bdc797733376bd')
docker: invalid spec: :/home/developer/azul: empty section between colons

Run 'docker run --help' for more information
make: *** [Makefile:238: format] Error 125
```

I believe this occurs because the script incorrectly assumes that if `/proc/self/mountinfo` exists, it must be running inside a container. I believe the bug was introduced by [this commit](https://github.com/DataBiosphere/azul/commit/6caccad62), which lines up roughly with my memory of when I started observing this issue.

Side note: the reason I didn't open this issue sooner is because it started occurring at around the same time I switched to my new computer, so I assumed it was a problem with my new environment.

--- a/scripts/resolve_container_path.py
+++ b/scripts/resolve_container_path.py
@@ -5,7 +5,7 @@ import sys
 
 import docker
 from more_itertools import (
-    one,
+    only,
 )
 
 from azul.logging import (
@@ -52,14 +52,17 @@ def resolve_container_path(container_path):
     except FileNotFoundError:
         log.info('Did not find %s', mountinfo_path)
     else:
-        api = docker.client.from_env().api
-        container_id = one(container_ids)
-        for mount in api.inspect_container(container_id)['Mounts']:
-            if container_path.startswith(mount['Destination']):
-                tail = os.path.relpath(container_path, mount['Destination'])
-                host_path = os.path.normpath(os.path.join(mount['Source'], tail))
-                log.info('Resolved %s to %s', container_path, host_path)
-                return host_path
+        container_id = only(container_ids)
+        if container_id is None:
+            log.info('Container ID not found in %s', mountinfo_path)
+        else:
+            api = docker.client.from_env().api
+            for mount in api.inspect_container(container_id)['Mounts']:
+                if container_path.startswith(mount['Destination']):
+                    tail = os.path.relpath(container_path, mount['Destination'])
+                    host_path = os.path.normpath(os.path.join(mount['Source'], tail))
+                    log.info('Resolved %s to %s', container_path, host_path)
+                    return host_path
     log.error('Failed to resolve container path %s', container_path)
     return None
 

